### ISSUE TITLE
  could not find java in ES_JAVA_HOME at /usr/lib/jvm/java-11-openjdk/bin/java

### ISSUE THREAD

**Issue by purple910** [SonarSource/docker-sonarqube#511]
docker run -d --name sonar1 -p 9000:9000 sonarqube:9.1.0-community 

Dropping Privileges
2021.11.10 11:24:22 INFO  app[][o.s.a.AppFileSystem] Cleaning or creating temp directory /opt/sonarqube/temp
2021.11.10 11:24:22 INFO  app[][o.s.a.es.EsSettings] Elasticsearch listening on [HTTP: 127.0.0.1:9001, TCP: 127.0.0.1:44326]
2021.11.10 11:24:22 INFO  app[][o.s.a.ProcessLauncherImpl] Launch process[[key='es', ipcIndex=1, logFilenamePrefix=es]] from [/opt/sonarqube/elasticsearch]: /opt/sonarqube/elasticsearch/bin/elasticsearch
could not find java in ES_JAVA_HOME at /usr/lib/jvm/java-11-openjdk/bin/java
2021.11.10 11:24:22 WARN  app[][o.s.a.p.AbstractManagedProcess] Process exited with exit value [es]: 1
2021.11.10 11:24:22 INFO  app[][o.s.a.SchedulerImpl] Waiting for Elasticsearch to be up and running
2021.11.10 11:24:22 INFO  app[][o.s.a.SchedulerImpl] Process[es] is stopped
2021.11.10 11:24:22 INFO  app[][o.s.a.SchedulerImpl] SonarQube is stopped

**Comment by tobias-trabelsi-sonarsource** [comment]
Hi @purple910 ,

see https://github.com/SonarSource/docker-sonarqube/issues/494 

tl;dr: either update your container runtime or start sonarqube with a modified seccomp profile
